packages/core/echo/echo-schema/src/expando.ts

```diff
@@ -6,13 +6,13 @@ import * as S from '@effect/schema/Schema';
 
 import { echoObject } from './annotations';
 
-export const ExpandoTypename = 'Expando';
+export const EXPANDO_TYPENAME = 'Expando';
```

Comment:
TYPE_EXPANDO = `dxos.org/type/Expando`
but i think we need to discuss typenames w/Dima.

Reply:
This is braking change